src/APICallManager.py

```diff
@@ -0,0 +1,38 @@
+import time
+
+from APICall import *
+
+key = os.environ['FRED_API_TOKEN_REGULATIONS_GOV']
+base_url = 'https://api.data.gov:443/regulations/v3/documents.json?'
+
+def api_call_manager(url):
+    pause = 0
+    while (pause < 51):
+        try:
+            document = call(url)
+            return document
+        except TemporaryException:
```

Comment:
Maybe reorder these 3 exceptions. Either shortest wait to longest, or vice versa